adafruit_msa3xx.py

```diff
@@ -30,7 +32,7 @@
 """
 
 __version__ = "0.0.0+auto.0"
-__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_MSA301.git"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython.git"
```

Comment:
I think this change was unintentional.
```suggestion
__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_MSA301.git"
```

Reply:
thanks. good catch. yah - global search/replace fail :(